crates/node/src/command.rs

```diff
@@ -37,7 +37,7 @@ impl SubstrateCli for Cli {
     fn load_spec(&self, id: &str) -> Result<Box<dyn sc_service::ChainSpec>, String> {
         Ok(match id {
             "dev" => {
-                let enable_manual_seal = self.sealing.map(|_| true);
+                let enable_manual_seal = self.run.sealing.map(|_| true);
```

Comment:
sealing is of type
```rust
#[derive(Debug, Copy, Clone, clap::ValueEnum, Default)]
pub enum Sealing {
    // Seal using rpc method.
    #[default]
    Manual,
    // Seal when transaction is executed.
    Instant,
}
```

Why do we ignore the `Instant` variant and make it a manual sealing anyway?

Reply:
Always have been like this. I just adapted it to work with our current code. @EvolveArt might be able to answer that question

Reply:
@EvolveArt do you know something?

Reply:
Seems to be linked to this [todo](https://github.com/keep-starknet-strange/madara/blob/main/crates/node/src/service.rs#L319)